string message

uint8 r
uint8 g
uint8 b

uint8 font_size